wca-guidelines.md

```diff
@@ -34,13 +34,6 @@ To be more informative, each Guideline is classified using one of the following
 
 ## <article-1><officials><officials> Article 1: Officials
 
-- 1c+) [CLARIFICATION] "The WCA Delegate" refers to anyone fulfilling the role of a WCA Delegate at a competition.
-- 1c3+) [RECOMMENDATION] Results should be ready at the end of the last day of competition.
```

Comment:
This could be merged with 1c4+) and added under 9.1.2. in WCA Competitions section of the WCA Competition Requirements Policy?